flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisProducer.java

```diff
@@ -207,23 +214,8 @@ public void invoke(OUT value) throws Exception {
 		if (this.producer == null) {
 			throw new RuntimeException("Kinesis producer has been closed");
 		}
-		if (thrownException != null) {
```

Comment:
nit: it would be easier to review the code, if refactor (like extracting this code to a method) was in separate commit then "real" "production" changes. Especially if those production changes are pretty minimal in term of number of changes line codes :)

Reply:
Will keep that in mind for the future :+1: